lib/file_reader.rb

```diff
@@ -0,0 +1,21 @@
+require 'json'
+
+class FileReader
```

Comment:
Maybe a module

```ruby
module FileReader
  def hash_from_txt(filename)
  end
  
  def hash_from_json(filename)
  end
end
```
```ruby
class Sentimental
  include FileReader
end
```

And if you want it as a class method somewhere else, just extend it.